executor/common_linux.h

```diff
@@ -2067,6 +2070,8 @@ static long syz_usbip_server_init(volatile long a0)
 	sprintf(buffer, "%d %d %s %d", port_num, client_fd, "0", speed);
 
 	write_file("/sys/devices/platform/vhci_hcd.0/attach", buffer);
+
+	close(client_fd);
```

Comment:
It would be nice to have some comment re. why we can close the client_fd here. Won't it break something?